### PR TITLE
Unify Soulflayer NM skill list IDs + remove skill lists from non-combat mobs

### PR DESCRIPTION
**Pull request by ibm2431**
(Look at `mob_skill_lists` first to understand the diff in `mob_pools`)

- Slip in a new skill list for NM Soulflayers right after the skill list for NQ Soulflayers
- Slide Spheroid, Spider, and "Structure" back by one
- Discover that "Structure" is a skill list for exoplates, but was also being assigned to random "structure" "mobs" which are non-combat. zero those skill lists out in their pools.
- Discover that although there isn't a skill list defined for them, pools for SOA reive "mob" structures (roots, nests, etc) have a skill list of 237 defined, zero them out
- Discovered that pool for Archaic Mirror (_not_ Rampart) was given skill list 238, despite it also being a non-combat structure, zero _that_ out too
- Finally, assign existing Soulflayer NMs' skill lists to be the new one we're slipping in after the NQs
- Delete duplicated Soulflayer NM skill lists and free up IDs

@kaincenteno 

**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date

### CODE REVIEW

File: sql/mob_skill_lists.sql [hunk 1]
```diff
@@ -1042,30 +1042,37 @@ INSERT INTO `mob_skill_lists` VALUES ('Snoll',232,1644);
...
+INSERT INTO `mob_skill_lists` VALUES ('Soulflayer_NM',234,1967); -- Tribulation
+INSERT INTO `mob_skill_lists` VALUES ('Soulflayer_NM',234,1968); -- Immortal Anathema
+-- INSERT INTO `mob_skill_lists` VALUES ('Soulflayer_NM',234,1969); -- Reprobation
+INSERT INTO `mob_skill_lists` VALUES ('Spheroid',235,561);
+INSERT INTO `mob_skill_lists` VALUES ('Spheroid',235,984);
+INSERT INTO `mob_skill_lists` VALUES ('Spider',236,810);
+INSERT INTO `mob_skill_lists` VALUES ('Spider',236,811);
+INSERT INTO `mob_skill_lists` VALUES ('Spider',236,812);
+INSERT INTO `mob_skill_lists` VALUES ('Structure_Exoplate',237,990);
+INSERT INTO `mob_skill_lists` VALUES ('Structure_Exoplate',237,991);
```
Comment (TeoTwawki) [inline sql/mob_skill_lists.sql:1064]: `Structure_Exoplate`? Shouldn't that be just `Exoplate`?

Comment (TeoTwawki) [reply sql/mob_skill_lists.sql:1064]: other than that all gravy :+1:

Comment (ibm2431) [reply sql/mob_skill_lists.sql:1064]: I was desperately trying to avoid it no longer being in alphabetical order 💦

Comment (TeoTwawki) [reply sql/mob_skill_lists.sql:1064]: prefix all entries with `A_`, everythings in alphabetical order, problem solved! :smile:

**Review by 59blargedy — COMMENTED** [review]
which is worse, having it out of numerical order or out of alpha?

**Pull request merged**